src/track/track.cpp

```diff
@@ -146,7 +146,7 @@ void Track::importMetadata(
         // stamps if file tags need to be re-imported.
         if (!metadataSynchronized.isNull()) {
             modified |= compareAndSet(
-                    &m_record.refMetadataSynchronized(),
+                    m_record.ptrMetadataSynchronized(),
```

Comment:
This is the only application that I had in mind and that actually makes sense to me.